.github/workflows/main-sync.yml

```diff
@@ -0,0 +1,22 @@
+name: Sync main branch
+
+env:
+  GITHUB_TOKEN: ${{ secrets.APACHE_SYNC_MIDSTREAM_TOKEN }}
+
+on:
+  schedule:
+    - cron:  '0 1 * * 0' 
+  workflow_dispatch:
+
+jobs:
+  sync-main:
+    name: Sync main branch
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Create a PR to sync main branch with main-apache
+        uses: kiegroup/kie-ci/.ci/actions/main-sync@main
+        with:
+          main_sync_workflow_exclude_paths: ${{ vars.MAIN_SYNC_WORKFLOW_EXCLUDE_PATHS }}
```

Comment:
you will add those variables vales to https://github.com/kiegroup/kogito-examples/settings/variables/actions, right? Same as you did for Drools

Reply:
Yes, I added the repo variables now